exercises/practice/binary-search/.docs/introduction.md

```diff
@@ -0,0 +1,9 @@
+# Introduction
+
+You have stumbled upon a group of mathematicians who are also singer-songwriters.
+They have written a song for each of their favorite numbers, and, as you can imagine, they have a lot of favorite numbers.
```

Comment:
Should we be more explicit in which meaning of "number" we mean?   When I first read this, I took it as the musical number, rather than a numeric number, even having seen that they were mathematicians doing this.

Then I had to think about what it meant, was it referring to the ditties that they liked, or their favorite scores, or how did they separate those arrangements.  Then, and only after, did I realize that in this context, we must mean numeric numbers, not musical numbers.

For some reason the music subject took over the context of the vocabulary used here.  Even after having read about numeric numbers in this review in the instructions file.

Reply:
We're going to need to make this change upstream, rather than here. I will open a PR tomorrow!

Reply:
Yeah, for both this and the other comment about emphasing the word, they'd need to happen upstream not here. I don't think the emphasising one would be worth repropogating across all 65 tracks as that's lots of noise for maintainers, but I think this one might be.

Reply:
If we take care of both at the same time, since the change is coming, would it be an appropriate time for it?  Perhaps in two commits, so that if a track wanted one but not the other, they could choose to do so.

Reply:
The changes from problem-specifications get added to the tracks by using configlet sync. They get all of the changes or none of the changes.

I won't be making hand-crafted PRs (or even lovingly scripted PRs) to allow people to cherry-pick an emphasis that does not meaningfully change the instructions.

I feel quite strongly that the only good reason to not accept a PR that syncs from problem-specs is if the instructions **contradict** the exercise in that track.

Reply:
@kotp I have opened the PR here: https://github.com/exercism/problem-specifications/pull/2226

If you like, feel free to suggest a PR to change the emphasis.

There is another PR open to the same exercise as well, so I am happy to wait to put in the new PRs to the tracks until all three PRs have been merged.

Reply:
Opened a [PR][pr] for the emphasis.  (Also, yes, the asterisks for bold rather than italics for emphasis, @iHiD .

[pr]:  exercism/problem-specifications#2227